capa/features/extractors/base_extractor.py

```diff
@@ -24,6 +25,29 @@
 # the feature extractor from which they were created.
 
 
+@dataclass
+class SampleHashes:
+    md5: str
+    sha1: str
+    sha256: str
+
+    def __iter__(self) -> Iterator[str]:
+        yield self.md5
+        yield self.sha1
+        yield self.sha256
```

Comment:
is this needed?

Reply:
I believe so, since when i tried doing something like `md5, sha1, sha256 = SampleHashes` it gave out an error of: `TypeError: cannot unpack non-iterable a object`